<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelFile1">
        <property name="text">
         <string>File 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file1Edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton1">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelFile2">
        <property name="text">
         <string>File 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file2Edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton2">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="plotButton">
      <property name="text">
       <string>Plot Curves</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>